lib/Ed25519VerificationKey2020.js

```diff
@@ -177,16 +182,22 @@ export class Ed25519VerificationKey2020 extends LDKeyPair {
         'Export requires specifying either "publicKey" or "privateKey".');
     }
     const exportedKey = {
+      '@context': Ed25519VerificationKey2020.SUITE_CONTEXT,
```

Comment:
Is this going to create trouble with existing APIs? Do we need a `includeContext` param?

Reply:
@dlongley I had the same question.  I'm find with an `includeContext` param that defaults to `false`.

Reply:
@dlongley I can't think of where it would get us in trouble. But a param sounds fine.

Reply:
It will probably get us into trouble anywhere we're composing a DID doc, for example:

https://github.com/digitalbazaar/did-method-key-js/blob/aa4dabcf390142654bed36bcdcd72c5f971eca15/lib/DidKeyDriver.js#L126-L127

It would inject `@context` where it was not there before.

Reply:
Right, which would be one thing if we made a breaking release, but I'm going to go ahead and add the flag now.

Reply:
Done: fe7a857dc529c048255eefa8be884dc601512ca2